utils/stringUtils.js

```diff
@@ -1,42 +1,20 @@
-/**
- * Returns a standardized version of the word
- * @param 	{string} 	word 	word to standardize
- * @returns 	{string} 		standardized value
- */
-const wordsStandardize = (word) => word ? word.trim().toLowerCase().normalize('NFD').replace(/[\u0300-\u036f]/g, '') : 'null';
-
-/**
- * Returns the correct boolean value for a parameter
- * @param 	{string} 	word 	parameter received
- * @returns 	{boolean} 		resulting boolean value
- */
+
+const wordsStandardize = (word) => word ? word.trim().toLowerCase().normalize('NFD').replace(/[\u0300-\u036f]/g, '') : null;
```

Comment:
you can't just look at one file when changing stuff, if you check the history of that line, you can see i already fixed it like you did, but that caused other problems which would take more work to fix, so I made it return an empty string which, in my opinion, is also perfectly fine

Reply:
also check the commit right after and read the message to see why i changed it back to return a string not null

Reply:
JSdocs are meaningless comments because they don't provide any additional information. Developers use it to generate API docs in HTML pages. To get clean code only convention can say to developers what the code does. Read Martin C clean codebook.